.circleci/base_config.yml

```diff
@@ -233,6 +242,8 @@ jobs:
       s3-prefix:
         type: string
         default: ""
+      arch:
```

Comment:
Which are possible? What is default?

Reply:
this is generated by `generate_config.py`; it will depend on the current target architecture. 
Its string is later on adjusted to download the go aux-binaries.